src/items/itemsLandingPage.component.tsx

```diff
@@ -193,409 +151,361 @@ function ItemsLandingPage() {
-                        Purchase Order Number
-                      </Typography>
-                      <Typography
-                        align="left"
-                        color="text.secondary"
-                        sx={{ wordWrap: 'break-word' }}
-                      >
-                        {itemData.purchase_order_number ?? 'None'}
-                      </Typography>
-                    </Grid>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <Typography align="left" color="text.primary">
-                        Warranty End Date
-                      </Typography>
-                      <Typography align="left" color="text.secondary">
-                        {itemData.warranty_end_date
-                          ? formatDateTimeStrings(
-                              itemData.warranty_end_date,
-                              false
-                            )
-                          : 'None'}
-                      </Typography>
-                    </Grid>
 
-                    <Grid item xs={12} sm={6} md={4}>
-                      <Typography align="left" color="text.primary">
-                        Delivered Date
-                      </Typography>
-                      <Typography align="left" color="text.secondary">
-                        {itemData.delivered_date
-                          ? formatDateTimeStrings(
-                              itemData.delivered_date,
-                              false
-                            )
-                          : 'None'}
-                      </Typography>
-                    </Grid>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <Typography align="left" color="text.primary">
-                        Is Defective
-                      </Typography>
-                      <Typography align="left" color="text.secondary">
-                        {itemData.is_defective ? 'Yes' : 'No'}
-                      </Typography>
-                    </Grid>
-
-                    <Grid item xs={12} sm={6} md={4}>
-                      <Typography align="left" color="text.primary">
-                        Usage Status
-                      </Typography>
-                      <Typography
-                        align="left"
-                        color="text.secondary"
-                        sx={{ wordWrap: 'break-word' }}
-                      >
-                        {itemData.usage_status}
-                      </Typography>
-                    </Grid>
-                    <Grid item xs={12} sm={6} md={4}>
-                      <Typography align="left" color="text.primary">
-                        System
-                      </Typography>
-                      <Typography
-                        align="left"
-                        color="text.secondary"
-                        sx={{ wordWrap: 'break-word' }}
+            <TabView<CatalogueLandingPageTabValue>
```

Comment:
Again not keen on this syntax, it is explicit, but when removed it just takes T as string anyway which would save a lot of the type conversions. The URL part wouldn't work if it wasn't a string internally.